app/assets/javascripts/my_modules/repositories.js

```diff
@@ -179,11 +240,152 @@ var MyModuleRepositories = (function() {
+      };
+    } else {
+      delete SELECTED_ROWS[id];
+    }
+
+    $(row).toggleClass('selected');
+
+    if (Object.keys(SELECTED_ROWS).length) {
+      $('#assignRepositoryRecords, #updateRepositoryRecords').attr('disabled', false);
+    } else {
+      $('#assignRepositoryRecords, #updateRepositoryRecords').attr('disabled', true);
+    }
+
+    refreshSelectAllCheckbox();
+  }
+
+  function renderFullViewActionButtons() {
+    var toolbar = fullViewModal.find('.dataTables_wrapper .toolbar');
+    toolbar.empty();
+    if (fullViewModal.data('rows-count') === 0) {
+      toolbar.append($('#my-module-repository-full-view-assign-button').html());
+    } else {
+      toolbar.append($('#my-module-repository-full-view-update-button').html());
+    }
+  }
+
+  function updateFullViewRowsCount(value) {
+    fullViewModal.data('rows-count', value);
+    fullViewModal.find('.assigned-repository-title').attr('data-rows-count', `[${value}]`);
+  }
+
+  function initRepoistoryAssignView() {
+    $('.repositories-dropdown-menu').on('click', '.repository', function(e) {
+      var assignUrlModal = $(this).data('assign-url-modal');
+      var updateUrlModal = $(this).data('update-url-modal');
+      var repositoryNameObject = $(this).find('.name').clone().addClass('assigned-repository-title');
+
+      fullViewModal.find('.repository-name').html(repositoryNameObject);
+      fullViewModal.data('rows-count', $(this).data('rows-count'));
+      fullViewModal.modal('show');
+      $.get($(this).data('table-url'), (data) => {
+        fullViewModal.find('.modal-body').html(data.html);
+        fullViewModal.data('assign-url-modal', assignUrlModal);
+        fullViewModal.data('update-url-modal', updateUrlModal);
+        renderFullViewTable(fullViewModal.find('.table'), { assign_mode: true });
+      });
+      e.stopPropagation();
+    });
+
+    fullViewModal.on('click', '.table tbody tr', function() {
+      var checkbox = $(this).find('.repository-row-selector');
+      checkbox.prop('checked', !checkbox.prop('checked'));
+      selectFullViewRow(this);
+    }).on('click', '.table tbody tr .repository-row-selector', function(e) {
+      selectFullViewRow($(this).closest('tr')[0]);
+      e.stopPropagation();
+    }).on('click', '#assignRepositoryRecords', function() {
+      openAssignRecordsModal();
+    }).on('click', '#updateRepositoryRecords', function() {
+      openUpdateRecordsModal();
+    });
+
+    updateModal.on('click', '.downstream-action', function() {
+      submitUpdateRepositoryRecord({ downstream: true });
+    }).on('click', '.task-action', function() {
+      submitUpdateRepositoryRecord({ downstream: false });
+    });
+  }
+
+  function openUpdateRecordsModal() {
+    var updateUrl = fullViewModal.data('update-url-modal');
+    $.get(updateUrl, { selected_rows: SELECTED_ROWS }, function(data) {
+      var assignList;
+      var assignListScrollbar;
+      var unassignList;
+      var unassignListScrollbar;
```

Comment:
Is it true?

Reply:
When you initialize scrollbar it assume, that you assign it to some variable. Which in future can be used to working with this scrollbar update and etc. In our case we don't update it after currently.  If we will not assign object to variable it will be also issue.